ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-0.3.yaml

```diff
@@ -0,0 +1,177 @@
+        source_path: /go/bin/bloatingcow
+  to: knative-serving-test-bloatingcow
+- dockerfile_path: openshift/ci-operator/knative-test-images/controller/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/controller
+  to: knative-serving-test-controller
+- dockerfile_path: openshift/ci-operator/knative-test-images/environment/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/environment
+  to: knative-serving-test-environment
+- dockerfile_path: openshift/ci-operator/knative-test-images/failing/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/failing
+  to: knative-serving-test-failing
+- dockerfile_path: openshift/ci-operator/knative-test-images/helloworld/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/helloworld
+  to: knative-serving-test-helloworld
+- dockerfile_path: openshift/ci-operator/knative-test-images/httpproxy/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/httpproxy
+  to: knative-serving-test-httpproxy
+- dockerfile_path: openshift/ci-operator/knative-test-images/observed-concurrency/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/observed-concurrency
+  to: knative-serving-test-observed-concurrency
+- dockerfile_path: openshift/ci-operator/knative-test-images/pizzaplanetv1/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/pizzaplanetv1
+  to: knative-serving-test-pizzaplanetv1
+- dockerfile_path: openshift/ci-operator/knative-test-images/pizzaplanetv2/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/pizzaplanetv2
+  to: knative-serving-test-pizzaplanetv2
+- dockerfile_path: openshift/ci-operator/knative-test-images/printport/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/printport
+  to: knative-serving-test-printport
+- dockerfile_path: openshift/ci-operator/knative-test-images/singlethreaded/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/singlethreaded
+  to: knative-serving-test-singlethreaded
+- dockerfile_path: openshift/ci-operator/knative-test-images/timeout/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/timeout
+  to: knative-serving-test-timeout
+promotion:
+  cluster: https://api.ci.openshift.org
+  namespace: openshift
+  name: knative-v0.3
+tests:
+- as: e2e
+  commands: "make test-e2e"
+  openshift_ansible_src:
```

Comment:
We want Openshift 4 and AWS so this needs to be different: `openshift_installer_src` and `cluster_profile: aws`. See this: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-0.2__v4.0.yaml#L127

Reply:
And after this the Prow jobs need to be regenerated, IMO.